src/clib/lib/job_queue/lsf_driver.cpp

```diff
@@ -926,15 +919,9 @@ void lsf_driver_set_bjobs_refresh_interval(lsf_driver_type *driver,
 }
 
 static void lsf_driver_shell_init(lsf_driver_type *lsf_driver) {
-    lsf_driver->last_bjobs_update = time(NULL);
+    lsf_driver->last_bjobs_update = time(nullptr);
```

Comment:
lsf_driver is always shell, so I guess we should inline this now.

Reply:
Agreed. Lets do that.